integration/spark/src/test/scala/org/apache/carbondata/geo/GeoTest.scala

```diff
@@ -112,6 +238,23 @@ class GeoTest extends QueryTest with BeforeAndAfterAll with BeforeAndAfterEach {
       result)
   }
 
+  test("test insert into non-geo table select from geo table") {
```

Comment:
please add a test case of insert into geo table, where insert rows will not have geo data. but select *  shows geo data

Reply:
modified existing test case and added validation for the geo column.